middleware/metadata.go

```diff
@@ -1,10 +1,10 @@
 package middleware
```

Comment:
for some reason, in previous ep20 commits, i modified this `middleware/metadata.go` file which shouldnt have been modified at all. these modifications resulted in breaking changes in middleware signatures. so in this commit im reverting it to its original state (prior to any ep20 commits)